client/landing/stepper/declarative-flow/internals/analytics/record-design.ts

```diff
@@ -114,6 +114,8 @@ export function getDesignEventProps( {
 		design_type: design.design_type,
 		is_premium: design.is_premium,
 		is_externally_managed: design?.is_externally_managed,
+		is_woo:
+			( design.software_sets || [] ).filter( ( set ) => set?.slug === 'woo-on-plans' ).length > 0,
```

Comment:
We originally had this prop already in `Design` type, but somehow got deleted here: https://github.com/Automattic/wp-calypso/pull/84483/files#diff-6e6af36e6ae5503714f992f94faa3ac3d5b61e3a87c3ccd6bad1e456d412c876L98.

Would be nice to move the logic back there, so that this function is only responsible for tracking. But it's non-blocking!

Reply:
Sounds good 👍

Reply:
I think I understand why it got removed... the abstraction hook is located in `/client/my-sites/theme/hook` but the original logic is in `/packages/data-store`, which is why it can't be used directly.

Furthermore, it's odd that Stepper and `/client/components` is now import from `/client/my-sites/theme/hook` https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20useBundleSettingsByTheme&type=code 🙁

Reply:
I've added the old logic back in the meantime. In the future we might want to consider moving the `useBundleSettings` hook to a package for it to be usable for other packages as well, such as `@automattic/data-store` or `@automattic/design-picker`! cc: @renatho @daledupreez @Automattic/t-rex

Reply:
Hi folks! 👋

The context is that it was removed to handle bundles in a generic way. The reason is that now it supports any software set.

The cases that used the removed property were changed to just check if it has a software set instead of if it's a WooCommerce software set. See the following points:

- https://github.com/Automattic/wp-calypso/pull/84483/files#diff-b87656ec14fd5239b510f6030f98085e83e609b1b820aa55226b55746eb74b26R36
- https://github.com/Automattic/wp-calypso/pull/84483/files#diff-a539d272b24cf98cb319fb27637a06c9faf52cfea40d0af87cf1b7c2fec0915cR157
- https://github.com/Automattic/wp-calypso/pull/84483/files#diff-a539d272b24cf98cb319fb27637a06c9faf52cfea40d0af87cf1b7c2fec0915cR390

More context: p6rkRX-75y-p2